src/java/htsjdk/samtools/util/ComparableTuple.java

```diff
@@ -0,0 +1,20 @@
+package htsjdk.samtools.util;
+
+/**
+ * Created by farjoun on 1/17/16.
+ */
+public class ComparableTuple<A extends Comparable<A>, B extends Comparable<B>> extends Tuple<A, B> implements Comparable<ComparableTuple<A, B>> {
+
+    public ComparableTuple(final A a, final B b) {
+        super(a, b);
```

Comment:
This should probably reject nulls unless you want to have the `compareTo` deal with them.